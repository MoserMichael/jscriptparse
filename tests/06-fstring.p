a = 2
b = 3

# combining strings
println(join([ "product: ", a * b , " sum product of squares: ", (a *a) + (b * b) ]))

# the same as an f-string
println("product: {a * b} sum product of squares: { (a *a) + (b * b) }")
