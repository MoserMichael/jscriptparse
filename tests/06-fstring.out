product: 6 sum product of squares: 13
product: 6 sum product of squares: 13
